join-chat-to-teams-meeting/Chat Experience/client.js

```diff
@@ -57,51 +43,72 @@ async function init() {
 
   callAgent = await callClient.createCallAgent(tokenCredential);
   callButton.disabled = false;
-
   chatClient = new ChatClient(endpointUrl, new AzureCommunicationTokenCredential(token));
 
   console.log("Azure Communication Chat client created!");
 }
 
 init();
 
+const joinCall = (urlString, callAgent) => {
+  const url = new URL(urlString);
+  console.log(url);
+  if (url.pathname.startsWith("/meet")) {
```

Comment:
Do you know if _all_ links are being replaced?
To be consistent it would be best if calling SDK, no matter the link type, always returned `call.callInfo.threadId` (which maybe is their plan -- but would need to check)

Otherwise we'll need:
- If link has threadId --> extract it (i.e. keep the old logic)
- If link is shortUrl --> get from the callInfo object

Reply:
We want to always use the call; and be rid of the thread ID from URL entirely.